README.md

```diff
@@ -49,6 +49,18 @@ this may prompt you for sudo access.
     grunt install
     grunt uninstall
 
+## Building a RPM package
+
+We use Docker to build the RPM package in a clean environment, this also avoids
+to polute your environment.
+
+To build a RPM package run the following command at the root of the git repository:
```

Comment:
How would this work for someone using OS X or Windows? Should they use a Linux VM for this task? Those details should be explained here.